drivers/src/persistent.rs

```diff
@@ -65,7 +65,6 @@ impl PersistentData {
                 memory_layout::MEASUREMENT_LOG_ORG
             );
             assert_eq!(addr_of!((*P).fuse_log) as u32, memory_layout::FUSE_LOG_ORG);
-            assert_eq!(P.add(1) as u32, memory_layout::BOOT_STATUS_ORG);
```

Comment:
We still need to assert the size of the struct to make sure there's no overlap. How about:

```
assert_eq!(P.add(1) as u32, memory_layout::FUSE_LOG_ORG + memory_layout::FUSE_LOG_SIZE);
```

Reply:
Done.